tests/api/search/search_entities.test.js

```diff
@@ -61,6 +64,13 @@ describe('search:entities', () => {
         })
       })
 
+      it('should not return results that do not contain the exact searched words', async () => {
```

Comment:
what about using the same wording as the reverse test right above this?
```suggestion
      it('should not return results that do not includes exact matches of each words', async () => {
```
[non-blocking]